aten/src/ATen/native/mkldnn/Gelu.cpp

```diff
@@ -0,0 +1,89 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/Config.h>
+
+
+#if !AT_MKLDNN_ENABLED()
+
+namespace at { namespace native {
+
+Tensor mkldnn_gelu(const Tensor& input) {
+  TORCH_CHECK(false, "mkldnn_gelu: ATen not compiled with MKLDNN support");
+}
+
+Tensor mkldnn_gelu_backward(const Tensor& grad_output, const Tensor& input, const Scalar& threshold) {
+  TORCH_CHECK(false, "mkldnn_gelu_backward: ATen not compiled with MKLDNN support");
+}
+
```

Comment:
merge `mkldnn_gelu_cpu` and `mkldnn_gelu_backward_cpu` into `mkldnn_gelu` and `mkldnn_gelu_backward`

Reply:
Done.